examples/bri-3/src/bri/utils/mapper.ts

```diff
@@ -0,0 +1,100 @@
+import { getType, Type, PropertyInfo, TypeKind } from 'tst-reflect';
+import { AccessModifier } from './access-modifiers';
+
+/**
+ * Generic mapper to convert object to Type at runtime
+ *
+ * @author : Manik-Jain
+ */
+export default class Mapper {
+  /**
+   * type convert from provided input to provided Type
+   * provides capabilities to define some explicit
+   * properties on the type
+   *
+   * Be careful with providing opts obeject as
+   * this will change the properties on the type
+   *
+   * @param from
+   * @param classType
+   * @param opts
+   * @returns generated object
+   */
+  static map<T>(from: any, classType: Type, opts?: any) {
+    if (!from) throw new Error('undefined source');
+
+    const result = {};
+    const mismatchedKeys = [];
+    if (classType.kind === TypeKind.Interface) {
+      const keys = classType.getProperties().map((param) => param.name);
+      Object.keys(from).forEach((prop) => {
+        if (keys.includes(prop)) {
+          result[prop] = from[prop];
+        } else {
+          mismatchedKeys.push(prop);
+        }
+      });
+
+      if (opts) {
+        Object.keys(opts).forEach((prop) => {
+          if (keys.includes(prop)) {
+            result[prop] = opts[prop];
+          } else {
+            mismatchedKeys.push(prop);
+          }
+        });
+      }
+
+      console.log(`Type conversion successful`);
```

Comment:
no need to log success as there will be a lot of them

Reply:
I think this will be helpful, specifically in debugging scenarios.